### DISCUSSION TITLE
File upload not working on mobile browser

### DISCUSSION THREAD

**Discussion by becca0330** [livewire/livewire#4910] [1. Help]
I just followed the basic upload example from [livewire doc](https://laravel-livewire.com/docs/2.x/file-uploads#basic-upload).
It works fine on laptop browser, but if I open the page from mobile browser, select a file and press save button, the file is still null.
Can you please help? :)

Enviroment:
Laravel v8.82
Livewire v2.10.1

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by joshhanley** [comment]
@becca0330 can you share what mobile OS and browser you are using? Thanks!

**Comment by satoved** [comment]
Ok, it's always about the file size. Things to check if you encounter in again:
1) File uploading can return "413 Request Entity Too Large" so check you Nginx/php settings for that
2) Check livewire.temporary_file_upload.rules config, which is 12MB by default and change if needed
3) If you use spatie/laravel-media-library, also check media-library.max_file_size config

Remember, that Livewire sends base64 encoded file contents, which is much larger than the original file.

**Comment by shinsenter** [comment]
Upload is not triggered on Mobile Safari (iPhone 14 Pro Max, iOS 16).
It works well on PC browsers!

**Reply by dominik-eller** [reply]
any workaround? I am having the same issue

**Reply by nuex001** [reply]
same
